Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -376,10 +376,6 @@ public void InitUCElementDetailsLocatorsGrid()
                 if (mWindowExplorerDriver.IsPOMSupported())
                 {
                     xUCElementDetails.InitLocatorsGridView();
-                    if (mContext.Platform == ePlatformType.Web)
```

Comment:
why this if was removed, we need to show Friendly locators tab only for web age

Reply:
on window explorerpage also we are not getting friendly locator element so no need of it because of that it's removed